README.md

```diff
@@ -60,6 +60,7 @@ your component.
     - Fields are derived from dialog properties (see below)
 - `options.content-exporter`: whether to configure sling model for content export
 - `options.model-adaptables`: array of adaptables to include in the Sling Model ('request' and/or 'resource')
+- `generate-in-html`: generate dialog fields in the html file
```

Comment:
should be `options.generate-in-html`

also, would something like `options.html-content` be more clear and consistent?  Perhaps put it right next to `options.html` for better organization

Reply:
Also, please add this config the to the demo `data-config.json` file, set to `false` by default, so that people realize the feature is there more easily.

Reply:
Renamed. Its better like that.
Added to json.